src/main/webapp/app/entities/administrator/administrator-update.tsx

```diff
@@ -111,62 +126,59 @@ export class AdministratorUpdate extends React.Component<IAdministratorUpdatePro
                     className="form-control"
                     name="role"
                     value={(!isNew && administratorEntity.role) || 'CC_HEAD'}
+                    required
                   >
                     <option value="CC_HEAD">{translate('clubmanagementApp.AdministratorRole.CC_HEAD')}</option>
                     <option value="VICE_CC_HEAD">{translate('clubmanagementApp.AdministratorRole.VICE_CC_HEAD')}</option>
                     <option value="SECRETARY">{translate('clubmanagementApp.AdministratorRole.SECRETARY')}</option>
                     <option value="TEASURER">{translate('clubmanagementApp.AdministratorRole.TEASURER')}</option>
                   </AvInput>
                 </AvGroup>
-                <AvGroup>
-                  <Label id="statusLabel" for="administrator-status">
-                    <Translate contentKey="clubmanagementApp.administrator.status">Status</Translate>
-                  </Label>
-                  <AvInput
-                    id="administrator-status"
-                    type="select"
-                    className="form-control"
-                    name="status"
-                    value={(!isNew && administratorEntity.status) || 'ACTIVE'}
```

Comment:
if you remove this input then what will be status of the administrator created?
null? or in Server is default to ACTIVE?

Reply:
no, because i see the figma there dont have this so i put active when form submit, will add it back

Reply:
@CarlosSia just add status for update
create default to ACTIVE is ok

Reply:
resolved by https://github.com/Five-Fishes/Club-Management-React-Client/pull/156/commits/94c83b6bfa3404d4bbc3c3fa0d1f7ebfec0137e2